File: CoreJava/accountpolymorphism-app/src/com/techlabs/account/CurrentAccount.java
package com.techlabs.account;

public class CurrentAccount extends Account {

	public CurrentAccount(int accno, String name, double amount) {
		super(accno, name, amount);
	}

	@Override
	public void withdraw(double amount) {
		if (this.getBalance() >= 1000 && this.getBalance() - amount >= 1000)
			this.balance = this.balance - amount;

	}

}

File: DotNET/C#/AccountPolymorphism/AccountPolymorphism/CurrentAccount.cs
using System;
namespace AccountPolymorphism
{
    class CurrentAccount : Account
    {
        public CurrentAccount(int accountnum, int balance, string name) : base(accountnum, balance, name)
        {
        }

        public override void withdraw(int amount)
        {
            if (Balance >= 1000 && Balance - amount >= 1000)
                Balance = Balance - amount;
            else
                Console.WriteLine("Withdraw failed.Minimum Balance is less");
        }
    }
}
